web/templates/_base.html

```diff
@@ -18,153 +18,67 @@
-      .box {
-      border: solid 1px #999;
-      border-left: solid 6px red;
-
-      width: 300px;
-      padding: 5px;
-      padding-left: 10px;
-      margin: 10px;
-      margin-left:0;
-      margin-right: 20px;
-      background: #eee;
-      }
-      .box h2 {
-        margin-top: 0;
-      }
-      #topbar {
-      width: 98%;
-      font-size: 120%;
-      margin-left: 0;
-      margin-right: 0;
-      }
-
-      .fullwidth {
-        width: 98%;
-        margin-left: 0;
-      margin-right: 0;
-      }
-
-      .top {
-      display: flex;
-      flex-direction: row;
-      flex-wrap: wrap;
-      }
-
-      div.content {
-      margin-left: 1em;
-      margin-right:1em;
-      font-family: monospace;
-      white-space: pre;
-      }
-
-      div.attachment {
-      margin-top: 10px;
-      margin-right:1em;
-      }
-      div.attachment::before {
-      content: "📎";
-      }
-
-      div.transaction {
-      padding: 10px;
-      margin-bottom: 10px;
-      border: solid 1px grey;
-      }
-      div.transaction.type-Status {
-      background: #FFE;
-      }
-
-      div.theader {
-      width: 98%;
-      background: #BBB;
-      display: inline-flex;
-      flex-direction: row;
-      flex-wrap: wrap;
-      margin-bottom: 10px;
-      padding: 3px;
-      }
-      div.theader div {
-      padding-right: 10px;
-      }
-      div.theader .email {
-      font-weight: 800;
-      }
-
-      /* Page stuff */
-      div#pheader {
-      width: 98%;
-      padding: 5px;
-      background: blue;
-      color: white;
-      font-weight: 900;
-      margin-bottom: 1em;
-      clear: both;
-      height: 3em;
-      }
-
-      .box#pheader2 {
-        border-left: solid 6px blue;
-        font-weight: 900;
-        font-size: 150%;
-        width:98%;
-      }
-
-      div#pfooter {
-      width: 98%;
-      padding: 5px;
-      background: blue;
-      color: white;
-      margin-top: 1em;
-      clear:both;
-      min-height: 3em;
-      }
-
-      a.button {
-        background: buttonface;
-        text-decoration: none;
-        border: solid 1px #999;
-        color: black;
-        padding: 4px;
-      }
-ul.tickets {
-  list-style-type: none;
-  padding-left: 2em;
-  text-indent: -2em;
-}
-      li.ticket {
-      }
-      .ticket a {
-        color: black;
-        text-decoration:none;
-      }
-      .ticket a:hover {
-        color: #777777;
-      }
-
-      .box#pfooter2 {
-        border-left: solid 6px green;
-        width:98%;
-      }
-    </style>
-    <meta name="robots" content="noindex, nofollow">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-    <title>{{ template "Title" . }} | Perlbug Archive</title>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+      <meta name="description" content="">
+      <meta name="author" content="">
+      <link rel="icon" href="{{ .Prefix }}/static/favicon.ico">
```

Comment:
I can't comment on the favicon file itself, but it's going to be very tiny.  We can change it later.